src/components/common/Marquee.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const Marquee: React.FC = () => {
+  const content = (
+    <>
+      <span className="mx-4 inline-block">EVENT PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">COMMERCIAL PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">PRODUCT PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">WEDDING PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">LANDSCAPE PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">BRANDING PHOTOGRAPHY</span>
+      <span className="mx-4 inline-block">PORTRAIT PHOTOGRAPHY</span>
+    </>
+  );
```

Comment:
why do you have it as a separate variable?

Reply:
? how to do it differently